src/code/game.c

```diff
@@ -85,19 +85,14 @@ void GameState_Draw(GameState* gameState, GraphicsContext* gfxCtx) {
     Graph_BranchDlist(polyOpa, nextDisplayList);
     POLY_OPA_DISP = nextDisplayList;
 
-    // Block prevents reordering, if(1) around the above block don't seem to help unlike in OoT
-    {
-        s32 requiredScopeTemp;
+    CLOSE_DISPS(gfxCtx);
 
-        Debug_DrawText(gfxCtx);
-    }
+    Debug_DrawText(gfxCtx);
```

Comment:
Nice catch. Just checked debug and the closedisps is actually before the DrawText. Nice